### PR TITLE
Feat: Multiple page template PDP - CMS Structure

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

- [x] Removes singleton.
- [x] Adds PDP template content type structure.  

## How it works?

Use the `vendemo` account, and now you will be able to see this template structure under the settings tab of hCMS.
Screenshot 2024-02-06 at 17 27 24
Screenshot 2024-02-06 at 16 16 35
the way we are going to deal with this content in frontend will be addressed in another task and the details are under the Multiple template page RFC.

See the response from hCMS API.
https://vendemo.myvtex.com/admin/new-cms/faststore/pdp

the other task will also cover the documentation.

### References

Doc PR
- https://github.com/vtex/faststore/pull/2236
[RFC Multiple page template details](https://docs.google.com/document/d/1qQ573B9iVadSrqjvjSt8ExDu7-uxa3_COXlKvqkTQ8Q)

### CODE REVIEW

File: packages/core/cms/faststore/content-types.json [hunk 1]
```diff
@@ -99,8 +99,27 @@
...
+          {
+            "name": "template",
+            "schema": {
+              "title": "Template",
+              "type": "object",
+              "properties": {
+                "value": {
+                  "title": "PDP template value (e.g. Slug: /apple-magic-mouse/p OR /department/category/subcategory/)",
+                  "type": "string",
+                  "description": "The value can be: the PDP slug template (e.g. /apple-magic-mouse/p). A /department/category/subcategory/ template (e.g. /office/ OR /office/chairs/ OR /office/chairs/classic). If this field is empty this template will be the generic PDP."
```
Comment (eduardoformiga) [inline packages/core/cms/faststore/content-types.json:115]: IMO we can improve these descriptions. Also the documentation should cover these details.

**Review by hellofanny — APPROVED** [review]

File: packages/core/cms/faststore/content-types.json (hunk 1 above)
Comment (lucasfp13) [inline packages/core/cms/faststore/content-types.json:115]: What about...?
```suggestion
                  "description": "Possible values: the PDP slug template (e.g. '/apple-magic-mouse/p'); A '/department/category/subcategory/' template (e.g. '/office/' OR '/office/chairs/' OR '/office/chairs/classic/'). If empty, this template will be the generic PDP."
```

**Review by lucasfp13 — APPROVED** [review]

File: packages/core/cms/faststore/content-types.json (hunk 1 above)
Comment (Gmantiqueira) [reply packages/core/cms/faststore/content-types.json:115]: Maybe follow the same pattern? Assuming that the user knows the page definitions
```suggestion
                  "description": "Possible values: the PDP slug template (e.g. '/apple-magic-mouse/p'); A PLP template (e.g. '/office/' OR '/office/chairs/' OR '/office/chairs/classic/'). If empty, this template will be the generic PDP."
```

**Review by Gmantiqueira — APPROVED** [review]

File: packages/core/cms/faststore/content-types.json (hunk 1 above)
Comment (hellofanny) [reply packages/core/cms/faststore/content-types.json:115]: I like mantis suggestion to keep the pattern, but I also think an example would be good! `office/chairs/classic`

Comment (eduardoformiga) [reply packages/core/cms/faststore/content-types.json]: Thank you, folks!

Let's keep some real examples in the docs, then.
- https://github.com/vtex/faststore/pull/2236

**Review by lucasfp13 — APPROVED** [review]

**Review by lariciamota — APPROVED** [review]

**Pull request merged**